utils/user_agent.tsx

```diff
@@ -103,6 +105,11 @@ export function isMobile(): boolean {
     return isIos() || isAndroid();
 }
 
+// returns true if and only if the user is using Mattermost from a PC with a small viewport size of <= 768px
```

Comment:
Is this really the intent of the ticket? We want global header to display on desktop and mobile web if you are on a phone, but NOT on small viewports on desktop?

It seems like if we want to hide on mobile we should hide in all cases. (What was happening before with the CSS)

Reply:
oh shoot ... you are absolutely right. I somehow mixed that up. I will change that.